op-challenger/flags/flags.go

```diff
@@ -108,6 +108,12 @@ var (
 		EnvVars: prefixEnvVars("CANNON_SNAPSHOT_FREQ"),
 		Value:   config.DefaultCannonSnapshotFreq,
 	}
+	GameWindowFlag = &cli.UintFlag{
```

Comment:
urfave CLI has a specific time duration flag which does nice parsing for different ways of specifying time

Reply:
Awesome, good callout